webapp/src/components/backstage/playbook_edit.tsx

```diff
@@ -526,6 +538,29 @@ const PlaybookEdit: FC<Props> = (props: Props) => {
                                         }}
                                     />
                                 </SidebarBlock>
+                                {experimentalFeaturesEnabled &&
+                                    <SidebarBlock>
+                                        <BackstageSubheader>
+                                            {'Retrospective Reminder Interval'}
+                                            <BackstageSubheaderDescription>
+                                                {'Reminds the channel at a specified interval to fill out the retrospective.'}
+                                            </BackstageSubheaderDescription>
+                                        </BackstageSubheader>
+                                        <StyledSelect
+                                            value={retrospectiveReminderOptions.find((option) => option.value === playbook.retrospective_reminder_interval_seconds)}
+                                            onChange={(option: { label: string, value: number }) => {
+                                                setPlaybook({
+                                                    ...playbook,
+                                                    retrospective_reminder_interval_seconds: option ? option.value : option,
```

Comment:
Is this different than the following?
```suggestion
                                                    retrospective_reminder_interval_seconds: option?.value,
```